src/chains/ethereum/ethereum/tests/api/eth/getBlockByNumber.test.ts

```diff
@@ -0,0 +1,99 @@
+import assert from "assert";
+import EthereumProvider from "../../../src/provider";
+import getProvider from "../../helpers/getProvider";
+import compile from "../../helpers/compile";
+const DEFAULT_DIFFICULTY = 1;
+let provider: EthereumProvider;
+
+// helper to mine an arbitrary number of blocks using dummy transactions
+const mineBlocks = async (number: number, provider: EthereumProvider) => {
+  const [from] = await provider.send("eth_accounts");
+  for (let x = number; x > 0; x--) {
+    await provider.send("eth_subscribe", ["newHeads"]);
+
+    await provider.send("eth_sendTransaction", [{ from, to: from }]);
+
+    await provider.once("message");
```

Comment:
If you don't need the transaction itself, you can do `await provider.send("evm_mine")` instead.

Oh, actually... now that #793 is merged (as of yesterday), you can do `await provider.send("evm_mine", [{blocks: number}])`, no (backwards :smile:) loop needed!

Reply:
Oh cool! I'll merge in `develop` and update this!

Reply:
I like deleting code.